src/js/utils/menu.js

```diff
@@ -83,4 +89,9 @@ export const setMenuState = () => {
     $serviceRequestsIndex.show();
     $registerLink.show();
   }
+
+  $supportLink.on("click", () => {
+    const currentHref = $supportLink.attr("href");
+    $supportLink.attr("href", currentHref + document.location.href);
```

Comment:
```suggestion
    $supportLink.attr("href", currentHref + document.location.href);
    $supportLink.attr("target", "_blank");
```

Reply:
I wonder how this will work when installed on the home screen? Also, on mobile, will the default not be to open the mail application? I doubt it will ever open In another browser tab. Also, if we _do_ set this, we will then also need to set `rel="noopener"`

Reply:
Good points, I have no idea how this would behave when installed on the home screen, and you're right that it would most likely open the mail app on mobile, so this really just affects desktop which is not a priority.

Let's ship without this